std/file.d

```diff
@@ -584,6 +584,133 @@ version(Windows) private void writeImpl(const(char)[] name, const(FSChar)* namez
     }
 }
 
+/** Definitions of common Byte Order Marks
+*/
+enum BOM {
+    ascii,    /// []
```

Comment:
Probably better to call this `unknown` or `none`. `ascii` is misleading, the value simply means that no BOM is present, but it could still be any kind of non-ASCII encoding.